include/qemu/bswap.h

```diff
@@ -1,6 +1,7 @@
 #ifndef BSWAP_H
 #define BSWAP_H
 
+#include "osdep.h"
```

Comment:
Is this necessary change? It's preferable to keep unrelated changes to the minimum, so that updating the QEMU base would be easier.